linode/manager
# fix: [M3-7571] - Missing label for Full Account Access Toggle

## Description 📝
This PR addresses the issue of the "Full Account Access" toggle missing a proper label. The changes in this PR improve accessibility in two ways. First by implementing correct markup semantics. Second, by ensuring screen readers inform the user more adequately by enhancing the output that is read.

## Changes  🔄
List any change relevant to the reviewer.
- Separates the "Full Account Access" toggle from the "User Permissions" header to improve readability.
- Changes the `variant` property for some `Typography` components to ensure proper `Header` levels.
- Update e2e tests

## Target release date 🗓️
09/30/2024

## Preview 📷
| Before  | After   |
| ------- | ------- |
| ![m3-7571-screenreader-before](https://github.com/user-attachments/assets/eef372e2-fcc0-486f-bbaf-f5b2b61c892c) | ![m3-7571-screenreader-after](https://github.com/user-attachments/assets/5ba33ee4-8739-4226-9c0c-ac8ac5599b6b) |
| ![m3-7571-desktop-before](https://github.com/user-attachments/assets/6faa16e9-8101-476e-b3d1-ba5d589f70b6) | ![m3-7571-desktop-after](https://github.com/user-attachments/assets/0a595c68-1241-4af1-a968-d0283aa728fa) |
| ![m3-7571-mobile-before](https://github.com/user-attachments/assets/e0090352-d4d6-44af-a4af-0e092173c0b1) | ![m3-7571-mobile-after](https://github.com/user-attachments/assets/fd51f353-a23e-412f-9a29-87bed97fdc4e) |

## How to test 🧪

### Prerequisites
(How to setup test environment)
- Visit the User Permissions landing page and activate the VoiceOver Utility (⌘ + F5) on your MAC computer.

### Verification steps
(How to verify changes)
- Use your keyboard to navigate to the "Full Account Access" toggle and changed the value a few times.
- Pay attention to the audio output of the VoiceOver utility and/or inspect the text transcript to verify the expected output.
- Verify that all Permission-related functionality works as expected.
- Verify that there are no unexpected visual regressions.


## As an Author I have considered 🤔
*Check all that apply*
